include/logging/log_msg2.h

```diff
@@ -28,7 +28,7 @@ extern "C" {
 #define LOG_MSG2_DEBUG 0
 #define LOG_MSG2_DBG(...) IF_ENABLED(LOG_MSG2_DEBUG, (printk(__VA_ARGS__)))
 
-#if CONFIG_LOG_TIMESTAMP_64BIT
+#ifdef CONFIG_LOG_TIMESTAMP_64BIT
```

Comment:
these unrelated changes seem fine, but are they necessary for the missing const?

Reply:
yes, you could split those into separate commit.

Reply:
Done